client/src/locales/fr-FR.json

```diff
@@ -1375,7 +1375,8 @@
                 },
                 "status": "Statut",
                 "download": "Télécharger",
-                "noInvoices": "Vous n'avez pas encore de factures."
+                "noInvoices": "Vous n'avez pas encore de factures.",
+                "retrieveError": "Impossible de récupérer la liste des factures."
```

Comment:
In this you placed this in "clientArea.dashboard.invoices", in the previous one in "clientArea.invoices" only. (see 3rd comment)
You need to change it for french file.